Intersect.Client/Core/Controls/Controls.cs

```diff
@@ -85,6 +85,7 @@ public void ResetDefaults()
             CreateControlMap(Control.OpenDebugger, Keys.F2, Keys.None);
             CreateControlMap(Control.OpenAdminPanel, Keys.Insert, Keys.None);
             CreateControlMap(Control.ToggleGui, Keys.F11, Keys.None);
+            CreateControlMap(Control.OpenGuild, Keys.G, Keys.None);
```

Comment:
I'm not sure if this determines the order of the controls or not but we might want this above the OpenSettings line as well?

Reply:
I'm not sure either but for the sake of clarity in the source files we might as well.